jgrapht-core/src/main/java/org/jgrapht/IncidentEdgeAccessor.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * (C) Copyright 2017-2017, by Dimitrios Michail and Contributors.
+ *
+ * JGraphT : a free Java graph-theory library
+ *
+ * This program and the accompanying materials are dual-licensed under
+ * either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1
+ * as published by the Free Software Foundation, or (at your option) any
+ * later version.
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by
+ * the Eclipse Foundation.
+ */
+package org.jgrapht;
+
+/**
+ * A vertex incident edges accessor. Incident edge accessors are helpful in creating algorithms
```

Comment:
Maybe we need to be a bit more verbose in the description. At least, based on the current description, I wouldn't really know what to do with this interface. Here's one attempt.

An accessor for edges incident to a vertex. Incident edge accessors are helpful in creating algorithms
which operate on both directed and undirected graphs. Through the accessor, one can iterate over a (subset of) the edges incident to a particular vertex; the exact implementation of the accessor is often dependent on the underlying graph (directed or undirected).